schema/openeox_core_json_schema.json

```diff
@@ -0,0 +1,63 @@
+{
+  "$schema": "https://json-schema.org/draft/2020-12/schema",
+  "$id": "https://docs.oasis-open.org/openeox/tbd/openeox_core_json_schema.json",
```

Comment:
I still find path separators more convincing:
`https://docs.oasis-open.org/openeox/tbd/schema/openeox-core.json`

Reply:
Should we always use the latest stage or according to what work product stage we are in? The latter would ensure that previous documents still maintain valid, the first avoids missed changes regarding links...

Reply:
Like with XML remote schema references I think some local editing (CATALOG in XML) is always in order /expected unfortunately. So I would keep it simple and locked into the main version (the "tbd" part of our path).
There is no optimal solution, I fear. This simplification promises to work out of the box towards the future for normal users.

Reply:
addressed in [b57311f](https://github.com/oasis-tcs/openeox/pull/10/commits/b57311f3895a5208794ebec25f72fbf200fb4d3e)